debian/control

```diff
@@ -0,0 +1,19 @@
+Source: souffle
+Maintainer: Nathan Keynes <nathan.keynes@oracle.com>
+Section: misc
+Priority: optional
+Standards-Version: 3.9.5
+Build-Depends: autoconf, automake, bison, debhelper (>= 9), flex, libsqlite3-dev, openjdk-7-jdk | openjdk-8-jdk
```

Comment:
I'm thinking we probably need to depend on explicit versions of autotools and bison? (or >= at least; I know earlier versions of bison don't work